golos.emit/src/golos.emit.cpp

```diff
@@ -98,47 +96,44 @@ void emission::emit() {
     }
```

Comment:
shouldn't we use "elapsed > _1e6_ * config::emit_interval" in the assert above?

Reply:
you're right, thx